rocAL_pybind/amd/rocal/fn.py

```diff
@@ -493,6 +493,36 @@ def resize_crop(*inputs, resize_width=0, resize_height=0, crop_area_factor=None,
         Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
     return (crop_resized_image)
 
+def resize_crop_fixed(*inputs, resize_width=0, resize_height=0, crop_w=None, crop_h=None, crop_pos_x=None, crop_pos_y=None,
```

Comment:
This is crop_Resize_fixed right?

Reply:
The random crop version of the CropResize node is added as resize_crop so this version is named resize_crop_fixed.